client/app/components/SelectItemsDialog.less

```diff
@@ -0,0 +1,9 @@
+.ant-list-item {
```

Comment:
This is unscoped, so it'll interact with all `ant-list-item` components. Is that intentional?

Reply:
Probably it's a behavior we can make use of elsewhere, but I'll scope it for now.